src/__tests__/parseWithPointers.ts

```diff
@@ -18,4 +20,111 @@ describe('json parser', () => {
 }`)
     ).toMatchSnapshot();
   });
+
+  test('parse complex', () => {
+    expect(
+      parseWithPointers(`{
```

Comment:
:trollface: - this seems to be one of the cases where the code formatted with Prettier is indeed worst than a manually indented one.

Reply:
Definitely... this should have been a fixture actually, since any time you prettify the code, the snapshot will change (different columns and most likely lines)